### DISCUSSION TITLE
Improve the import function

### DISCUSSION THREAD

**Discussion by Ruffyop** [gotson/komga#1617] [Ideas]
Hey, it would be great if the import feature could be improved with some additions:

1. Import new series (did not exist before), you could split the select series field to search for new series.
2. add a dropdown for existing series in addition to the search field
3. default selection in the settings for Hardlink - Move Files
4. the ability to select a default import folder (or use the last one used by default).

Thanks for this great software!

**Comment by gotson** [comment]
> add a dropdown for existing series in addition to the search field

unrealistic for libraries with thousands of series



> default selection in the settings for Hardlink - Move Files

the setting is already persisted on your local storage

> the ability to select a default import folder (or use the last one used by default).

it is already persisted in local storage.

**Reply by akhdanfadh** [reply]
Hi @gotson , how about the first question? Is there already a way to do that?

**Reply by Ruffyop** [reply]
> >add a dropdown for existing series in addition to the search field
>
> unrealistic for libraries with thousands of series
> 
Ok, that makes sense, but would it then be possible for the search to start much earlier? That is, when I enter B, I can already see the ads that start with B (e.g. Berserk) and not only when I have entered Ber

> > default selection in the settings for Hardlink - Move Files
> 
> the setting is already persisted on your local storage
>
I cannot find such a setting, can you tell me where it is?

> 
> > the ability to select a default import folder (or use the last one used by default).
> 
> it is already persisted in local storage.
>
I cannot find such a setting, can you tell me where it is?

>Import new series (did not exist before), you could split the select series field to search for new series.
>
Would that be possible? This is the function I miss the most.

**Reply by gotson** [reply]
> Ok, that makes sense, but would it then be possible for the search to start much earlier? That is, when I enter B, I can already see the ads that start with B (e.g. Berserk) and not only when I have entered Ber

you need at least 3 characters for the search to be relevant

> I cannot find such a setting, can you tell me where it is?

there's no visible setting, the value is stored in local storage for your browser

> Would that be possible? This is the function I miss the most.

this is not planned for the moment, Komga does not create new folders in your directory structure
